docs/proposals/fluentd-secure-forward.md

```diff
@@ -0,0 +1,35 @@
+# Allow Fluentd to secure forward to other Fluentd/Logstash (over ES-Copy)
```

Comment:
How automated do we need to make this do you think?  I think if we were going to have a flag it would have fluentd not send to an internal ES at all and only would forward / ES_COPY. And we could look at the configurations to know which one we were going to actually use?

Does this make sense to be in a configmap since Fluentd already uses a configmap for its normal configuration and we could just as easily describe how to update that?